drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -9,6 +9,7 @@
 #include <drivers/i2c.h>
 #include <init.h>
 #include <drivers/sensor.h>
+#include <drivers/sensor/bq274xx.h>
```

Comment:
Move this to `drivers/sensor/bq274xx/bq274xx.h`.

Reply:
Other public API extensions aren't nested in a folder either though...